app/src/main/java/com/example/dine_and_donate/Activities/SignUpActivity.java

```diff
@@ -89,11 +85,7 @@ public void onItemSelected(AdapterView<?> parent, View view, int position, long
                     mSignUpBtn.setVisibility(View.VISIBLE);
                     mPassword.setVisibility(View.VISIBLE);
                 } else {
-                    mName.setVisibility(View.GONE);
-                    mEmail.setVisibility(View.GONE);
-                    mPassword.setVisibility(View.GONE);
-                    mOrgPhone.setVisibility(View.GONE);
-                    mSignUpBtn.setVisibility(View.GONE);
+                    hideView();
```

Comment:
this was for the case when the user went from a selection of Consumer/Organization to no selection in the spinner (there should be no text views if the user has not selected the user type they are). I deleted it, since that probably would not happen.